code/game/objects/structures/artstuff.dm

```diff
@@ -77,7 +78,22 @@ GLOBAL_LIST_INIT(globalBlankCanvases, new(AMT_OF_CANVASES))
 		GLOB.globalBlankCanvases[whichGlobalBackup] = I
 		. = I
 
-
+/obj/item/canvas/examine(mob/user)
+	..()
+	if(in_range(src, user))
+		show(user)
+	else
+		to_chat(user, "<span class='warning'>You need to get closer to get a good look at this canvas!</span>")
+
+/obj/item/canvas/proc/show(mob/user)
+	var/icon/img = getFlatIcon(src)
```

Comment:
ree

Reply:
Tell me what you'd prefer.

Reply:
there's nothing that can be done to make this better I am just registering my protest into the night